src/main/scala/DemoFortis.scala

```diff
@@ -173,6 +181,66 @@ object DemoFortis {
       }
     }
 
+    if (mode.contains("tadaweb")) {
+      streamProvider.buildStream[TadawebEvent](ssc, streamRegistry("tadaweb")) match {
+        case Some(stream) => stream
+          .map(event => AnalyzedItem(
+            event,
+            source = event.tada.name,
+            analysis = Analysis(keywords =
+              keywordExtractor.extractKeywords(event.text) ::: keywordExtractor.extractKeywords(event.title)
+            )
+          ))
+          .filter(_.analysis.keywords.nonEmpty)
+          .map(fortisEvent => {
+            val language = languageDetection.detectLanguage(fortisEvent.originalItem.text)
+            val sentiment: Option[Double] = fortisEvent.originalItem.sentiment match {
+              case "negative" => Some(0)
+              case "neutral" => Some(0.6)
+              case "positive" => Some(1)
+              case _ => language match {
+                case Some(lang) =>
+                  if (supportedLanguages.contains(lang))
```

Comment:
If this is here to ensure that the sentiment detector can handle the language of the event, it already does this check for you, so no need to do it here ([source](https://github.com/CatalystCode/project-fortis-spark/blob/23b90f13750fff59bebb6f9d87346579b46b16df/src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/sentiment/SentimentDetector.scala#L16)). If we want to filter out events that aren't from our target languages, I'd say to do it as a separate step to increase clarity (so one `.map` to add the language followed by a `.filter` to remove the non-targeted events, followed by a `.map` to add the sentiment).